app/models/message.rb

```diff
@@ -0,0 +1,45 @@
+class Message < ActiveRecord::Base
+
+  self.inheritance_column = :_type_disabled
```

Comment:
What happened with message types? Can we remove this column?  Is it a good idea to drop its support?